command/job_run.go

```diff
@@ -208,7 +213,11 @@ func (c *JobRunCommand) Run(args []string) int {
 	}
 
 	if output {
-		req := api.RegisterJobRequest{Job: job}
+		req := struct {
+			Job *api.Job
+		}{
+			Job: job,
+		}
```

Comment:
need a class for deserializing a `Job: `-embedded job spec, without all of the other job registration options.